charts/argo-workflows/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v3.5.5
 name: argo-workflows
 description: A Helm chart for Argo Workflows
 type: application
-version: 0.40.12
+version: 0.40.13
```

Comment:
better to bump minor version ...? 🤔

Reply:
It shouldn't be a breaking change, so I think you're ok.